Webtest.sh

```diff
@@ -45,7 +45,7 @@ fi
 # Check if max payload set to value passed in cmd line parameter -maxpayloadsizekb
 SIZE=$($CURL "${BASE_URL}/echo?size=1048576" |wc -c)
 # It should return 8194, not sure why the difference
-if ! [ "$SIZE" -eq 8310 ]; then
+if [ "$SIZE" -lt 8194 ] || [ "$SIZE" -gt 8400 ]; then
```

Comment:
8*1024
8192

so make it 8191 for -lt ?

we should also check it doesn't compress but... some other time